scipy/stats/tests/test_hypotests.py

```diff
@@ -159,3 +160,253 @@ def test_callable_cdf(self):
         r1 = cramervonmises(x, distributions.beta.cdf, args)
         r2 = cramervonmises(x, "beta", args)
         assert_equal((r1.statistic, r1.pvalue), (r2.statistic, r2.pvalue))
+
+
+def assert_raises_with_match(exception_type, match, function, *args, **kwargs):
+    with assert_raises(exception_type, match=match):
+        function(*args, **kwargs)
```

Comment:
Instead of creating this function, let's just use the context manager in the tests.  I think the context manager makes a nicer looking test.